src/main/java/com/weaponcharges/WeaponChargesPlugin.java

```diff
@@ -214,15 +214,35 @@ private void enableDevMode()
 		eventBus.register(devtools);
 	}
 
+	private int lastDegradedHitsplatTick = -1000;
+
 	@Subscribe
 	public void onHitsplatApplied(HitsplatApplied e) {
-		if (e.getActor() != client.getLocalPlayer() || e.getHitsplat().getHitsplatType() != Hitsplat.HitsplatType.DAMAGE_ME) return;
-		ChargedWeapon crystalHelm = getEquippedChargedWeapon(EquipmentInventorySlot.HEAD);
-		if (crystalHelm != null) addCharges(crystalHelm, -1, false);
-		ChargedWeapon crystalBody = getEquippedChargedWeapon(EquipmentInventorySlot.BODY);
-		if (crystalBody != null) addCharges(crystalBody, -1, false);
-		ChargedWeapon crystalLegs = getEquippedChargedWeapon(EquipmentInventorySlot.LEGS);
-		if (crystalLegs != null) addCharges(crystalLegs, -1, false);
+		Hitsplat.HitsplatType hitType = e.getHitsplat().getHitsplatType();
+		ChargedWeapon helm = getEquippedChargedWeapon(EquipmentInventorySlot.HEAD);
+		if (helm == ChargedWeapon.SERPENTINE_HELM) {
+			if (hitType == Hitsplat.HitsplatType.DAMAGE_ME || hitType == Hitsplat.HitsplatType.BLOCK_ME) {
+				if (client.getTickCount() - lastDegradedHitsplatTick >= 90) {
```

Comment:
I think > is correct, not >=, unless you have testing that says otherwise?

Reply:
I've never seen a 90-tick gap, smallest I've seen is 91.

Reply:
I will remove the = and we can revisit this if desync is apparent as a result.